src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -2080,23 +2084,37 @@ type Place {
     """The stop related to the place."""
     stop: Stop
 
-    """The bike rental station related to the place"""
-    bikeRentalStation: BikeRentalStation @deprecated(reason: "Use vehicleRentalStation and rentalVehicle instead")
+    """
+    The sequence of the stop in the trip. This is not required to start from 0 or be consecutive - any
+    increasing integer sequence along the stops is valid.
+
+    The purpose of this field is to identify the stop within the pattern so it can be cross-referenced
+    between it and the itinerary. It is safe to cross-reference when done quickly, i.e. within seconds.
+    However, it should be noted that realtime updates can change the values, so don't store it for
+    longer amounts of time.
+
+    Depending on the source data, this might not be the GTFS `stop_sequence` but another value, perhaps
+    even generated.
+    """
+    stopSequence: Int
```

Comment:
We discussed in the dev meeting today if we should now consider the possibility of the place being between two stops in a trip that allows continuous stops. I think we should maybe call this `stopPosition` (seems like that term was used in the transmodel API already) and make it return an union of a StopPosition (I don't think it's possible to use Int in an union or interface so need a new type for this basic case) and ContinuousStopPosition (not sure what the naming should be) which has something like `after' and `before` (and perhaps later on something that tells more precisely where the stop happened. It's a bit unfortunate that we are implementing support for continuous stops in the schema before we have a support for them but I think we can't change the return type to an union later on and I think the union needs to have at least two entries (not sure about this). I think in the StopTime type we don't have to handle the continuous stop case as we don't return that type from an itinerary, right?

I don't want to dictate how this should be done especially when I'm putting the burden of implementation on @leonardehrenfried, so we can discuss this more if you have any criticism of what I've suggested.

Reply:
I've implemented this today.

Reply:
I've deviated a bit how the types are resolved (via an interface even though it's a GraphQL union). I hope that's ok.

Reply:
This allows me to not return `Object` in the fetcher.